invenio_grobid/utils.py

```diff
@@ -26,7 +26,89 @@
 
```

Comment:
In general. I agree. E.g. the upload hook is done like this already (in this file there is just a generic sample). We can add a similar hook for the conversion. However, I'd suggest provide a "generic" conversion here, out-of-the-box, that people can use.